plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Core/Instance/Flat.hs

```diff
@@ -57,7 +59,7 @@ tags and their used/available encoding possibilities.
 | Data type        | Function          | Bit Width | Total | Used | Remaining |
 |------------------|-------------------|-----------|-------|------|-----------|
 | default builtins | encodeBuiltin     | 7         | 128   | 54   | 74        |
-| Terms            | encodeTerm        | 4         | 16    | 8    | 8         |
+| Terms            | encodeTerm        | 4         | 16    | 10   | 6         |
```

Comment:
Are these comments actually helpful to anybody? It's a hassle to keep them up-to-date.